Katello/katello
# Fixes #36992 - Offer hint for enabling synced content option on host OS edit page

#### What are the changes introduced in this pull request?
The host edit page for the operating system now contains a tooltip popup that explains to users how to enable the 'Synced Content' option.

#### Considerations taken when implementing this change?
n/a

#### What are the testing steps for this pull request?

1. Create two different content views and lifecycle environments. Have one of these CV/LE pairs contain synced kickstart repos for the OS/architecture you will be using for your host in the next step.
2. Enter the host creation form. Before selecting an environment/LE/CV, go to the 'Operating System' tab and ensure the 'Synced Content' field is greyed out. Ensure you can click the tooltip.
3. Select both pairs of CV/LE for the host and ensure the 'Synced Content' field updates appropriately.
4. Create the host.
5. Edit the new host and ensure the 'Synced Content' field looks ok and the tooltip still functions.
6. Change the host to the other CV/LE. Edit the host and ensure the 'Synced Content' field looks ok and the tooltip still functions.
